sections/collection-list.liquid

```diff
@@ -31,7 +31,7 @@
     {% endunless %}
   
     <slider-component class="slider-mobile-gutter">
-      <ul class="collection-list grid grid--1-col{% if section.blocks.size < 5 %} grid--{{ section.blocks.size }}-col-tablet{% else %} grid--3-col-tablet{% endif %}{% if section.settings.swipe_on_mobile %} slider slider--tablet grid--peek{% endif %} collection-list--{{ section.blocks.size }}-items"
+      <ul class="collection-list grid grid--1-col{% if section.settings.collections_to_show_row == 1 %} grid-full-width{% else %} grid--{{ section.settings.collections_to_show_row }}-col-desktop{% endif %} grid--{{ section.blocks.size }}-col-tablet{% if section.settings.swipe_on_mobile %} slider slider--tablet grid--peek{% endif %} collection-list--{{ section.blocks.size }}-items"
```

Comment:
I kind of think a tablet setting would be getting a little too cryptic. Do merchants know they'd need to resize their browser width a bit to see the tablet setting change? Otherwise we could just choose a fixed number for tablet like we currently do (I'm fine with this). Or we add more magic and implement some conditions like `if desktop shows more than 4 per row then tablet shows 3, if desktop shows 4 or less then tablet shows 2`

Reply:
I dont think we should add any magic 🤔.  I think we should just find a number of rows that works well on tablet based on the amount of products/columns (_not_ the desktop setting)

The merchants wont see this too when editing since they cant see tablet: https://screenshot.click/07-51-23ed2-548gw.png

Reply:
I can see the logic in what Ken is saying. If the amount of items per row on tablet was to be set to 3 at all time, it might not be ideal. As if the merchant sets it to 2 per row on desktop then having 3 on tablet seem against the layout chosen on desktop. 

If it's two, it feel like it would potentially be too big sometimes. But the merchant could also choose 1 on desktop and then it looks different on tablet 🤷 

Tablet screen size will come up on small laptop or none full width browser windows as well 🤔

Reply:
To play devil's advocate, wed have to pick tablet to follow the logic of the settings from `mobile` or `desktop`.  I am not sure if the tablet should follow the setting for desktop or for mobile. This is why maybe it should be optimized for tablet depending on the section 🤔  The setting does say `desktop` and `mobile`, this is why I think it's pretty straight forward to the merchant, but maybe not. I can see this going both ways 😓 

cc @wiktoriaswiecicka what do you think?